### ISSUE TITLE
Code chunks accessibility issue "Link missing a text alternative"

### ISSUE THREAD

**Issue by josenino95** [carpentries/varnish#192]
This is my final issue on accessibility, related to #190 and #191.

The SiteImprove tool identified that code chunks weirdly generate hyperlinks that don't have alt text.

For example, look at [line 443 of this HTML source](view-source:https://swcarpentry.github.io/git-novice/04-changes.html) for an example with the Git SW lesson. Here an <a> tag is created, `href="#cb2-1"`, but no alt text. So that's why the accessibility tool is flagging it.

Do you know the reason why code chunks create these <a> tags? After a quick review, I see this happening to Shell and Python chunks, but not R or Output chunks.
